### DISCUSSION TITLE
Allow directus_fields table to be used in relations

### DISCUSSION THREAD

**Discussion by takuy** [directus/directus#17326] [Feature Requests]
I would like to have a collection that allows me to relate data specifically to other fields in other collections. Is it possible to have directus_fields available as an option here? 

![image](https://user-images.githubusercontent.com/1631095/215231880-0987d8df-80c2-4a08-8892-c9dd3fd37158.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
